apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/entity/blockchain/TransactionImpl.java

```diff
@@ -67,8 +55,8 @@
 import java.util.Map;
 import java.util.Set;
 
-import static com.apollocurrency.aplwallet.apl.core.transaction.AccountControl.SET_PHASING_ONLY;
-
+import static com.apollocurrency.aplwallet.apl.core.transaction.TransactionTypes.TransactionTypeSpec.SET_PHASING_ONLY;
+@Slf4j
```

Comment:
Duplicated log
@Slf4j
and
private static final Logger LOG = LoggerFactory.getLogger(TransactionImpl.class);

let's leave @Slf4j only.

Reply:
Fixed